File: Annadata/app/src/main/java/com/mananhirak/annadata/activityshelpers/HeyToaster.java
package com.mananhirak.annadata.activityshelpers;


import android.content.Context;
import android.view.Gravity;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import com.mananhirak.annadata.R;

public class HeyToaster {

   public static void HeyToaster1(Context context, String text, int Layout){
      Toast toast= new Toast(context);
      View view = LayoutInflater.from(context).inflate(Layout,null);
      toast.setView(view);
      toast.setGravity(Gravity.TOP,0,45);
      TextView textView=view.findViewById(R.id.toast_i_text);
      textView.setText(text);
      toast.setDuration(Toast.LENGTH_LONG);
      toast.show();
   }


}

File: Annadata/app/src/main/java/com/manan/annadata/activityshelpers/HeyToaster.kt
package com.manan.annadata.activityshelpers

import android.content.Context
import android.widget.Toast
import android.view.LayoutInflater
import android.view.Gravity
import android.widget.TextView
import com.manan.annadata.R

object HeyToaster {
    fun HeyToaster1(context: Context?, text: String?, Layout: Int) {
        val toast = Toast(context)
        val view = LayoutInflater.from(context).inflate(Layout, null)
        toast.view = view
        toast.setGravity(Gravity.TOP, 0, 45)
        val textView = view.findViewById<TextView>(R.id.toast_i_text)
        textView.text = text
        toast.duration = Toast.LENGTH_LONG
        toast.show()
    }
}